docs/docs/graphql-api.md

```diff
@@ -0,0 +1,242 @@
+
+```javascript
+export const pageQuery = graphql`
+  query HomePageQuery {
+    site {
+      siteMetadata {
+        description
+      }
+    }
+  }
+`
+```
+
+**Note**: the `const` that gets exported doesn't need to be called `pageQuery`, it can have anything, Gatsby just looks for an exported `graphql` string from the file.
+
+#### Returns
+
+A page query returns a data object passed into the props of the page component of the file the query is in.
+
+```javascript
+// highlight-start
+const HomePage = ({ data }) => {
+  // highlight-end
+  return (
+    <div>
+      Hello!
+      {data.site.siteMetadata.description} // highlight-line
+    </div>
+  )
+}
+```
+
+### `StaticQuery`
+
+Gatsby provides a `StaticQuery` component that can be used in any component to get data from GraphQL. You can only have one `StaticQuery` per page, in order to include the data you need from multiple sources you can use one query with multiple root fields. It cannot take variables as arguments.
+
+Also, refer to the [guide on querying data in components with static query](/docs/static-query/).
+
+#### Params
+
+The `StaticQuery` component takes two values as props:
+
+- query: a `graphql` query string
+- render: a component with access to the data returned
+
+```jsx
+<StaticQuery
+  query={graphql`
+    query HeadingQuery {
+      site {
+        siteMetadata {
+          title
+        }
+      }
+    }
+  `}
+  render={data => (
+    <header>
+      <h1>{data.site.siteMetadata.title}</h1>
+    </header>
+  )}
+/>
+```
+
+#### Returns
+
+StaticQuery returns data in a render prop:
+
+```jsx
+<StaticQuery
+  // ...
+  // highlight-start
+  render={data => (
+    <header>
+      <h1>{data.site.siteMetadata.title}</h1>
+    </header>
+  )}
+  // highlight-end
+/>
+```
+
+### `useStaticQuery`
+
+The `useStaticQuery` hook can be used similar to `StaticQuery` in any component or page, but doesn't require the use of a component and render prop.
+
+Because it is a React hook, the [rules of hooks](https://reactjs.org/docs/hooks-rules.html) apply and you'll need to be using React and ReactDOM version 16.8.0 or later. Because of how queries currently work in Gatsby, only one instance of `useStaticQuery` is supported in each file.
+
+Also, refer to the [guide on querying data in components with useStaticQuery](/docs/use-static-query/).
+
+#### Params
+
+The `useStaticQuery` hook takes one argument:
+
+- query: a `graphql` query string
+
+```jsx
+const data = useStaticQuery(graphql`
+  query HeaderQuery {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+  }
+`)
+```
+
+#### Returns
+
+The `useStaticQuery` hook returns data in an object:
+
+```jsx
+const data = useStaticQuery(graphql`
+  query HeaderQuery {
+    site {
+      siteMetadata {
+        title
+      }
+    }
+  }
+`)
+return (
+  // highlight-start
+  <header>
+    <h1>{data.site.siteMetadata.title}</h1>
+  </header>
+  // highlight-end
+)
+```
+
+## GraphQL Fragments
```

Comment:
```suggestion
## Query fragments
```
Adjusting to be consistent with the above headline change for `Query components and hooks`